### ISSUE TITLE
[BUG] Issue with a backspace key

### ISSUE THREAD

**Issue by mikeBashStuff** [aristocratos/bashtop#100]
**Describe the bug**
It seems like the `$backspace` char is not properly encoded. Currently, I am not able to change any `↲` friendly option by clearing contents of the `$inputting_value` with a backspace. However, if I do this:
```
diff --git a/bashtop b/bashtop
index e18db45..58f6c95 100755
--- a/bashtop
+++ b/bashtop
@@ -156,7 +156,8 @@ declare esc_character tab backspace sleepy late_update skip_process_draw winches
 declare -a disks_free disks_total disks_name disks_free_percent disks_io saved_key themes
 printf -v esc_character "\u1b"
 printf -v tab "\u09"
-printf -v backspace "\u7F"
+# printf -v backspace "\u7F"
+printf -v backspace "\u08"
 #printf -v enter_key "\uA"
 printf -v enter_key "\uD"
```
the key works. Based on the ascii table, `x7f` doesn't map to a `backspace` key.  If the intention is to actually use `DEL`, I think it would require the same treatment as any other `$esc_character` sequence. But in here, we basically fall through the `key=""` case when the `get_key()` is called.

From what I see, DEL is being actually detected as a `delete` key, but it's not used anywhere (or at least nothing explicitly looks it up). So something like this I guess could also do the trick:
```
diff --git a/bashtop b/bashtop
index e18db45..9be2ec6 100755
--- a/bashtop
+++ b/bashtop
@@ -3403,7 +3403,6 @@ get_key() { #? Get one key from standard input and translate key code to readabl

                #* Read 3 more characters if a leading escape character is detected
                if [[ $key == "${enter_key}" ]]; then key="enter"
-               elif [[ $key == "${backspace}" ]]; then key="backspace"
                elif [[ $key == "${tab}" ]]; then key="tab"
                elif [[ $key == "$esc_character" ]]; then
                        esc=1; key=$(stty -cooked min 0 time 0; dd bs=1 count=3 2>/dev/null); fi # read -rsn3 -t 0.001 key 2>/dev/null || true
@@ -3415,7 +3414,7 @@ get_key() { #? Get one key from standard input and translate key code to readabl
                                '[D'*) key="left" ;;
                                '[C'*) key="right" ;;
                                '[2~') key="insert" ;;
-                               '[3~') key="delete" ;;
+                               '[3~') key="backspace" ;;
                                '[H'*) key="home" ;;
                                '[F'*) key="end" ;;
                                '[5~') key="page_up" ;;
```
Taking for granted we want to use actual `DEL` as `backspace` (which I doubt that's the case :)).
**To Reproduce**
Go to options, try to modify, e.g.,  `Update ms` by inputting entirely new value.

**Expected behavior**
Each press of the `backspace` (or DEL) key should remove single char from the currently set string.

**Info**
 - Bashtop version: `0.8.24`
 - Linux distribution and version: `Debian 10`
 - Terminal used: `xterm`
 - Font used: `Terminal`
 - Bash version: `5.0.3(1)-release`
 - Locales: `en_US.UTF-8`

**Comment by aristocratos** [comment]
Huh, it's meant to be backspace. Didn't actually look the code up when I set up the variable, but captured it from my keyboard with `read`. And I haven't set any custom settings for backspace in the shell. Guess the default on my system was mapped to delete?

Might just as well add both for backspace then, since there is no string "editing" while typing enabled as of now anyways.

**Comment by aristocratos** [comment]
From https://www.tldp.org/HOWTO/Keyboard-and-Console-HOWTO-5.html

>         Linux and XFree86 come configured with both the Backspace and Delete
>         keys generating Delete.
> 
>         You can fix this by using any one of the xmodmap, xkeycaps, or
>         loadkeys programs to make the key in question generate the BackSpace
>         keysym instead of Delete.

So seems to be some legacy settings at play in Xorg maybe?

**Comment by aristocratos** [comment]
Fixed in v0.8.25
